### PR TITLE
FS-2199 Changing title to headings

### PR DESCRIPTION
**Pull request by adamdavies1**
Titles in management case are now headings for correct markup
image

### CODE REVIEW

**Review by gidsg — CHANGES_REQUESTED** [review]
This is incorrect, there should only one `<h1>` on the page, so that should be reserved for the main heading and others should be `<h2>` and below.

**Comment by adamdavies1** [comment]
> 

Should now be the correct h tags

File: form_jsons/public/en/upload-business-plan.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
...
...>\n<li>any supporting evidence that you will deliver the benefits to the community you have outlined</li>\n<li>planned skills or resources needed to manage the asset</li>\n<li>your experience of delivering similar projects</li>\n<li>any relevant project management expertise</li>\n<li>roles you plan to recruit to help you manage the asset</li>\n</ul>\n\n\n\n<h4 class=\"govuk-caption-m govuk-!-font-weight-bold govuk-!-padding-top-1\">Organisation details</h4>\n<ul class=\"govuk-list govuk-list--bullet govuk-caption-m\">\n<li>your organisation’s governance and membership structures, with supporting evidence</li>\n<li>members of your board, including their:</li>\n<ul class=\"govuk-list govuk-list--bullet govuk-caption-m\">\n<li>role</li>\n<li>main responsibilities</li>\n</ul>\n<li>copies of your organisation’s memorandum and articles of association</li>\n<li>a copy of your most recent governance board meeting minutes</li>\n<li>supporting evidence of how you will manage finances and prevent fraud. For example, copies of your policy documents</li>\n<li>supporting evidence of how you will manage performance and risks. For example, copies of your policy documents</li>\n</ul>\n\n\n</div>\n"
```
Comment (gidsg) [inline form_jsons/public/en/upload-business-plan.json:47]: @adamdavies1  I think there is already an `<h1>` on this page 
```
<h1 class="govuk-heading-l">
    Management case
  </h1>
```
So there shouldn't be another `<h1>` below that.

Comment (adamdavies1) [reply form_jsons/public/en/upload-business-plan.json]: These should now have to correct headings

**Review by gidsg — APPROVED** [review]

**Pull request merged**